### PR TITLE
feat: contribution guide

### PR DESCRIPTION
**Pull request by anatolykislov**
Add contribution guide according to #77

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+* Open a new issue in the [issue tracker](https://github.com/sourcerer-io/sourcerer-app/issues).
+* Fork and clone the repo with `git clone https://github.com/your-username/sourcerer-app.git`.
+* Create a new branch based off the `develop` branch.
+* Make changes.
+* Make sure all tests pass.
+* Submit a pull request, referencing any issues it addresses.
+
+We will review your Pull Request as soon as possible. Thank you for contributing!
+
+We value input from each member of the community, however we urge you to abide our [code of conduct](https://github.com/sourcerer-io/sourcerer-app/blob/master/CODE_OF_CONDUCT.md).
```
Comment (sergey-surkov) [inline CONTRIBUTING.md]: abide by

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+We welcome contributions to [Sourcerer App](https://github.com/sourcerer-io/sourcerer-app) by the community.
+
+See [README](https://github.com/sourcerer-io/sourcerer-app#build) for more information about prerequirements and building project.
+
+## How to report a problem
+
+Please search before creating a new issue. Feel free to add issues related to the app or [sourcerer](https://sourcerer.io) site.
+
+## Submitting сhanges
```
Comment (asurkov) [inline CONTRIBUTING.md]: How to write a patch is probably nicer

**Comment by dr-nafanya** [comment]
Thank you!
Do you have any special recommendation on integration testing of proposed changes? Or should one just use their personal account for testing?

File: CONTRIBUTING.md (hunk 2 above)
Comment (anatolykislov) [reply CONTRIBUTING.md]: I really like "Submitting сhanges". Other opinions?

**Pull request merged**